p/themes/Swage/swage.scss

```diff
@@ -603,6 +603,28 @@ form {
 	}
 }
 
+@supports (scrollbar-width: thin) {
+	#sidebar {
+		scrollbar-color: rgba(255, 255, 255, 0.05) rgba(0, 0, 0, 0.0);
+	}
+	#sidebar:hover {
+		scrollbar-color: rgba(255, 255, 255, 0.3) rgba(0, 0, 0, 0.0);
+	}
+}
+/* Note: This selector has to be split from the -webkit- prefixed equivalent below,
```

Comment:
The comments might not be needed anymore, since there is now a natural split due to the `@support {}` construction

Reply:
I think you're right!